Portal/vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/ConfigurationBusiness.java

```diff
@@ -106,8 +106,6 @@ public boolean validateSession(String email, String session) throws BusinessExce
             if (email != null && session != null) {
                 UserDAO userDAO = CoreDAOFactory.getDAOFactory().getUserDAO();
                 if (userDAO.verifySession(email, session) && !userDAO.isLocked(email)) {
-                    String newSession = UUID.randomUUID().toString();
-                    userDAO.updateSession(email, newSession);
```

Comment:
In my knowledge session ids can be used than once. The standard use is that the session id is unique and doesn't change for a single session (from login to logout).
In fact it is not done at each request in VIP, but only at each load (in each F5 in not in each AJAX request).

The best solutions to avoid session hijacking is to use SSL/TLS (done), and to regenerate the session id after the login (done in VIP).

To change the session id with each request is an other way to reinforce the security but is known to have side effects because concurrent calls can lead to session invalidation. This is exactly what happened in VIP in SOAP requests. Closely timed requests cause session id jam and authentication errors. Then after 5 requests the account is locked which is a major problem. This happened in selenium automatic tests.

Session id regeneration should be used along with a mechanism to avoid this concurrency problem. For instance remembering for a short period the old cookies but that implies some work.
I think it does not bring much value in VIP but has a major side-effect and it is best to remove it.

Some resources : 
- https://en.wikipedia.org/wiki/Session_hijacking#Prevention
- http://php.net/session_regenerate_id (PHP implementation, see warning)